vault/changelog.release.2022-08-02.md

```diff
@@ -0,0 +1,85 @@
+---
+id: ts7a4ba891ctryu35gxcc3m
+title: 0.106
+desc: ''
+updated: 1659450027598
+created: 1659372767472
+---
+
+Dendron 0.106 has sprouted  🌱
+August 02, 2022
+
+**Remove Delete Node and Show Preview commands** The `Delete Node` command is now replaced with the `Delete` command and `Show Preview` command is now replaced with the `Toggle Preview` command. Time to change your custom keyboard shortcuts to reflect the changes!
```

Comment:
If you've set custom keyboard shortcuts for these commands, a reminder to change them now!